packages/storage/src/indexed-db/index.ts

```diff
@@ -254,9 +256,17 @@ export class IndexedDb implements IndexedDbInterface {
   }
 
   // creates a local copy of the asset list from registry (https://github.com/prax-wallet/registry)
-  async saveRegistryAssets(assets: Metadata[]) {
+  async saveRegistryAssets(registryClient: ChainRegistryClient, chainId: string) {
+    const { commit } = registryClient.version();
+    const lastPosition = await this.db.get('REGISTRY_VERSION', 'commit');
+
+    // Registry version already saved
+    if (lastPosition === commit) return;
```

Comment:
An older registry commit can overwrite a newer one. But it seems this can never occur

Reply:
Yes this is true, but thinking this is not a likely scenario for us